src/Color.purs

```diff
@@ -160,22 +163,40 @@ rgba' r g b a = rgba (round $ r * 255.0)
 rgb' :: Number -> Number -> Number -> Color
 rgb' r g b = rgba' r g b 1.0
 
--- | Create a `Color` from hue, saturation, lightness and alpha values. The
+-- | Create a `Color` from Hue, Saturation, Lightness and Alpha values. The
```

Comment:
Any particular reason you want to change that to capitalized form? If so, let's please keep it consistent (hue is lowercase in the second sentence)